Formula/apache-arrow.rb

```diff
@@ -56,6 +58,7 @@ def install
       -DARROW_PLASMA=ON
       -DARROW_PROTOBUF_USE_SHARED=ON
       -DARROW_PYTHON=ON
+      -DARROW_S3=ON
```

Comment:
Why is it useful to enable this?

Reply:
It allows to read/write files from AmazonS3/Min.io cloud storage. Without this option error returned for s3 related functions in apache-arrow